runtimes/pythonrt/runner/main.py

```diff
@@ -210,27 +210,21 @@ def Execute(self, request: pb.ExecuteRequest, context: grpc.ServicerContext):
 
         return resp
 
-    def ActivityReply(
-        self, request: pb.ActivityReplyRequest, context: grpc.ServicerContext
-    ):
+    def ActivityReply(self, request: pb.ActivityReplyRequest, context: grpc.ServicerContext):
```

Comment:
Looks like we have a different limit on line length. For Python code we use `ruff format` without any modifications.

Reply:
fixed (i think)